src/main/java/org/apache/maven/plugin/compiler/AbstractCompilerMojo.java

```diff
@@ -184,6 +184,7 @@
      * 
      * <b>NOTE: </b>Since 3.8.0 the default value has changed from 1.5 to 1.6.
      * Since 3.9.0 the default value has changed from 1.6 to 1.7
+     * Since 3.11.0 the default value has changed from 1.7 to 1.8
```

Comment:
I would introduce in this place a list or line break - now everything is in one line 

https://maven.apache.org/plugins/maven-compiler-plugin/compile-mojo.html#source